From 5373a0030c401e3f75d6126c7c80716244e231c4 Mon Sep 17 00:00:00 2001
From: Florian Weisshardt <florian.weisshardt@ipa.fraunhofer.de>
Date: Mon, 22 Sep 2014 11:36:07 +0200
Subject: [PATCH] Workaround for bundled binaries

---
 rpm/template.spec.em | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/rpm/template.spec.em b/rpm/template.spec.em
index d56a4b5..2b32533 100644
--- a/rpm/template.spec.em
+++ b/rpm/template.spec.em
@@ -1,3 +1,5 @@
+%undefine _missing_build_ids_terminate_build
+
 Name:           @(Package)
 Version:        @(Version)
 Release:        @(RPMInc)%{?dist}
-- 
1.9.3

